mozilla/blurts-server
# MNTOR-978: Migrate breach resolution (part 3)

# References: 
Jira: MNTOR-978
Figma: n/a




# Description
After #2766 is merged and ran, the `breach_resolution` column should be updated to now use breachId as the key for resolutions

This PR modifies the way breach resolutions are retrieved to conform to the new key. Instead of using recency Index, we now have a flag to determine whether the breach resolution json has been migrated or not. If it has, we retrieve the json object and parse it based on breachId. If not, we fall back to using recency index. The GET is meant to be backwards compatible.

This PR also modifies the way breach resolutions are saved to the database by replacing `recencyIndex` with `breachId` 

# Screenshot (if applicable)

Not applicable.

# How to test
- After running `deprecateBreachResolve` and `recencyToBreachId` on your local db, try loading breaches page to test regression
- Try making a PUT call via `api/v1/breaches` and make sure the call returns a `200`


# Checklist (Definition of Done)
